### PR TITLE
Add title to widget indicating selected cell

### PR DESCRIPTION
**Pull request by dstansby**
This adds a title of the form (e.g. for id 59) `"Lineage tree #59"`. @KristinaUlicna is this a good title? Or do you have any better suggestion for it?

I also:
- removed the legend indicating this in the 1D property plotter
- changed the tree viewer labels to all be solid white, as the title now indicates which cell is selected (fixes https://github.com/lowe-lab-ucl/arboretum/issues/47)

Fixes https://github.com/lowe-lab-ucl/arboretum/issues/61.

### CODE REVIEW

File: napari_arboretum/plugin.py [hunk 1]
```diff
@@ -24,22 +25,28 @@ class Arboretum(QWidget, TrackPropertyMixin):
...
+        layout.addWidget(self.title, row, col, Qt.AlignHCenter)
+        self.title.setText("Arboretum")
+
+        # Add tree plotter
+        row = 1
         layout.addWidget(self.plotter.get_qwidget(), row, col)
         # Add property plotter
-        row = 1
+        row = 2
         layout.addWidget(self.property_plotter.get_qwidget(), row, col)
```
Comment (paddyroddy) [inline napari_arboretum/plugin.py:45]: Now that we've dropped support for `python=3.7` we could take advantage of the 🦭 operator and do 
```suggestion
        layout.addWidget(self.property_plotter.get_qwidget(), row:=2, col)
```

**Review by paddyroddy — APPROVED** [review]

**Comment by dstansby** [comment]
I've updated this with changes discussed in our meeting earlier today.

File: napari_arboretum/visualisation/matplotlib_plotter.py [hunk 2]
```diff
@@ -23,8 +23,6 @@ def draw_current_time_line(self, time: int) -> None:
         if not hasattr(self, "_mpl_time_line"):
             self._mpl_time_line: Line2D = self.axes.axvline(time, color="white")
         else:
-            # TODO: fix this
```
Comment (paddyroddy) [inline napari_arboretum/visualisation/matplotlib_plotter.py:26]: Did this get fixed? Not sure what it is

**Review by paddyroddy — APPROVED** [review]

File: napari_arboretum/visualisation/matplotlib_plotter.py (hunk 2 above)
Comment (dstansby) [reply napari_arboretum/visualisation/matplotlib_plotter.py:26]: Yeah, it's an old comment I forgot to delete in a previous PR, sorry for the confusion!

**Pull request merged**